code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -51,9 +51,9 @@ All effects don't start immediately, but rather get worse over time; the rate is
 		if(reac_volume >= 5)
 			var/obj/item/book/affectedbook = O
 			affectedbook.dat = null
-			to_chat(usr, "<span class='notice'>Through thorough application, you wash away [affectedbook]'s writing.</span>")
+			O.visible_message("<span class='notice'>\The [O]'s writing is washed away by [id]!</span>")
 		else
-			to_chat(usr, "<span class='warning'>The ink smears, but doesn't wash away!</span>")
+			O.visible_message("<span class='warning'>\The [O]'s ink is smeared by [id], but doesn't wash away!</span>")
```

Comment:
Perhaps we should add a message about the Codex Gigas not being erased?  Seeing as it operates in a non-book like manner, this won't erase it.